bodhi/server/consumers/masher.py

```diff
@@ -429,8 +430,13 @@ def work(self):
             self.check_all_karma_thresholds()
             self.obsolete_older_updates()
 
+            # Clean old composes after successful completion
```

Comment:
I think you can remove the `after successful completion` from the comment since this is not true any more.